Spigot-API-Patches/0301-Add-vs-code-files-to-.gitignore.patch

```diff
@@ -0,0 +1,48 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: dodison <kacpik@mapik.eu>
+Date: Mon, 17 May 2021 17:55:57 +0200
+Subject: [PATCH] Add vs code files to .gitignore
+
+
+diff --git a/.gitignore b/.gitignore
+index e431e3435737e28394d81b56568a08b3c3148b9b..87d2e932f6f2c524785012b7d12c76b65a648b92 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -5,13 +5,14 @@
+ 
+ # netbeans
+ /nbproject
+-/nb-configuration.xml
++nb*.xml
+ 
+ # we use maven!
+ /build.xml
+ 
+ # maven
+ /target
++dependency-reduced-pom.xml
+ 
+ # vim
+ .*.sw[a-p]
+@@ -22,6 +23,9 @@
+ /dist
+ /manifest.mf
+ 
++/world
```

Comment:
no idea why this needs to be in there, use `./paper t` instead of starting in the root of the project